pkgs/development/tools/continuous-integration/woodpecker/frontend.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, fetchFromGitHub, fetchYarnDeps, mkYarnPackage }:
+let
+  inherit (import ./common.nix { inherit lib fetchFromGitHub; })
```

Comment:
If we assign this to a variable and inherit from it we don't need to maintain two lists to inherit things.

Reply:
I guess you mean this:

```
inherit (import ./common.nix { }) a b c;

buildGoModule {
  # xyz
  inherit a b c;
}
```

to

```
common = inherit (import ./common.nix { });

buildGoModule {
  # xyz
  inherit (common) a b c;
}
```

---

I was also debating whether we should just have a general `commonGo` that holds stuff like `src` `version` `vendorSha256` `ldflags` etc and just do

```
inherit (import ./common.nix { }) commonGo meta;

buildGoModule (commonGo // {
  # xyz
  meta = meta // {
  };
})
```